templates/quiz/meta-information.php

```diff
@@ -7,7 +7,8 @@
  * @since 3.9.0
  * @since 4.0.0 Unknown.
  * @since 4.17.0 Return early if accessed without a logged in user or the quiz can't be loaded from the `$post` global.
- * @version 4.17.0
+ * @since [version] Added function for getting Quiz's questions count.
```

Comment:
Did you add a function here? You use a new function instead of...

Reply:
sorry, fixed now.